onnxruntime/core/providers/openvino/openvino_execution_provider.cc

```diff
@@ -297,6 +296,22 @@ void CheckGraphSupported(const onnxruntime::GraphViewer& graph_viewer, std::stri
       }
     }
 
+    if (node->OpType() == "Mul" || node->OpType() == "Add" || node->OpType() == "Div" || node->OpType() == "Sub") {
+      for (size_t i = 0; i < node->InputDefs().size(); i++) {
+        if (node->InputDefs()[i]->TypeAsProto()->tensor_type().elem_type() == ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_INT64) {
```

Comment:
Do you think we should have a logic to reject for all unsupported data types? I can see George asking why only int64, this is not generic.